Framework/include/TrainingNTupleVars.h

```diff
@@ -0,0 +1,72 @@
+#ifndef TRAININGNTUPLEVARS_H
+#define TRAININGNTUPLEVARS_H
+
+class TrainingNTupleVars
+{
+private:
+    std::string myVarSuffix_;
+
+    
+    void moreVars(NTupleReader& tr)
+    {
+        const auto& Jets                    = tr.getVec<TLorentzVector>(("Jets"+myVarSuffix_));
+        const auto& GoodJets_pt30           = tr.getVec<bool>("GoodJets_pt30"+myVarSuffix_);
+        const auto& GoodLeptons             = tr.getVec<std::pair<std::string, TLorentzVector>>("GoodLeptons"+myVarSuffix_);
+
+        
+
+
+
+
+
+
```

Comment:
Lots of spaces it would be cleaner to remove them. Normally I wouldn't care but I added a comment for another file anyway.